omegatrade-app/frontend/src/app/components/company/manage-company/manage-company.component.html

```diff
@@ -0,0 +1,55 @@
+<app-header></app-header>
+<app-sidebar></app-sidebar>
+<div class="spinner" *ngIf="this.loader">
+  <mat-spinner></mat-spinner>
+</div>
+<main>
+  <div class="container-fluid mt-20">
+    <div class="row">
+      <div class="col-6" style="margin: auto;">
+        <span style="font-size: 18px;">
+          Company List
+        </span>
+      </div>
+      <div class="col-6 table-options">
+        <mat-form-field>
+          <input matInput (keyup)="applyFilter($event.target.value)" placeholder="Search">
+        </mat-form-field>
+        <button mat-flat-button color="primary" class="add-company-btn" (click)="openCompanyDialog()">Add
+          Company</button>
+      </div>
+    </div>
+    <div class="row mt-10">
+      <div class="col-12">
+        <table mat-table [dataSource]="dataSource" matSort class="mat-elevation-z8">
+
+          <!-- Name Column -->
+          <ng-container matColumnDef="companyName">
+            <th mat-header-cell *matHeaderCellDef mat-sort-header> Company Name </th>
+            <td mat-cell *matCellDef="let element"> {{element.companyName}} </td>
+          </ng-container>
+
+          <!-- shortcode Column -->
+          <ng-container matColumnDef="companyShortCode">
+            <th mat-header-cell *matHeaderCellDef mat-sort-header> Short Code</th>
+            <td mat-cell *matCellDef="let element"> {{element.companyShortCode}} </td>
+          </ng-container>
+
+          <ng-container matColumnDef="action">
+            <th mat-header-cell *matHeaderCellDef mat-sort-header> Action</th>
+            <td mat-cell *matCellDef="let row" style="padding:5px;">
+              <a href="javascript:void(0)" class="edit-company-btn" (click)="this.openCompanyDialog(row)"
```

Comment:
`(click)="this.openCompanyDialog(row)"` this.openCompanyDialog will not work, this word should not be used anywhere in the html file

Reply:
this issue has been resolved.